src/shogun/multiclass/tree/C45ClassifierTree.cpp

```diff
@@ -766,11 +814,12 @@ void C45ClassifierTree::init()
 	m_certainty=SGVector<float64_t>();
 	m_types_set=false;
 	m_weights_set=false;
-
+	m_feature_importances = SGVector<float64_t>();
+	SG_ADD(
+	    &m_feature_importances, "feature_importances", "feature importances");
```

Comment:
This should either be registered with watch_method or pass the READONLY flag

Reply:
this ? `SG_ADD(
	    &m_feature_importances, "feature_importances", "feature importances",ParameterProperties::READONLY);`

Reply:
yes!

Reply:
i've suggested @LiuYuHui that actually maybe it'd be better to unify this for trees... meaning the feature importance...

Reply:
this pr or other pr?

Reply:
just do it (here)! (tm)